dist/src/main/java/io/camunda/identity/IdentityModuleConfiguration.java

```diff
@@ -40,4 +41,8 @@
       HibernateJpaAutoConfiguration.class,
       DataSourceTransactionManagerAutoConfiguration.class
     })
+@ConditionalOnProperty(
+    name = "zeebe.broker.gateway.enable",
+    havingValue = "true",
+    matchIfMissing = true)
```

Comment:
@tmetzke Sure! :) ... this was decided during a conversation in slack. Will pass you the link there in a sec ... the tl;dr of it is: Since I made IdentityModuleConfiguration part of StandaloneBroker in order for it to function when it's run with gateway active, this annotation ensures the IdentityModuleConfiguration is not present when gateway isn't active there.

Reply:
OK, nice, thanks! I am honestly still struggling to understand those different properties correctly 🙈 This is more of a learning question, I think 😄